src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,11 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
     public String getRandomColor() {
-        return null;
+        int index = new Random().nextInt(Colors.values().length);
+        Colors color = Colors.values()[index];
+        return String.valueOf(color);
```

Comment:
It's better to use `name()` instead of `String.valueOf()` for getting the String representation of enum constants to avoid issues if `toString()` is overridden in the enum.

Reply:
fix comment